tools/cockpit.spec

```diff
@@ -502,7 +502,7 @@ fi
 test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 
 # check for deprecated PAM config
-if grep --color=auto pam_cockpit_cert %{_sysconfdir}/pam.d/cockpit; then
+if test -f %{_sysconfdir}/pam.d/cockpit/pam_cockpit_cert*; then
```

Comment:
But this does something completely different. There is no such thing as a  /etc/pam.d/cockpit/ directory, with some pam_cockpit_cert file. That is an *entry* in the /etc/pam.d/cockpit *file*.

/etc/pam.d/cockpit exists at this point, as it's part of the rpm and this is the `%post` script. You may have deleted it, but then your cockpit installation is rather broken: You will not be able to login locally (remote logins should still work). OOI, why did you do that, do you want to support a bastion host use case only?

So, we could append a `2>/dev/null`, or do a `test -f %{_sysconfdir}/pam.d/cockpit && grep ..`.